pb/node.proto

```diff
@@ -86,41 +85,53 @@ message GetPeersResponse {
     repeated string peerIDs = 1;
 }
 
-// ConnectRequest is used to connect to libp2p peers
-message ConnectRequest {
-    // a slice of all multiaddrs we want to connect to
-    repeated string multiAddrs = 1;
+// CONNMGMTREQTYPE indicates the particular ConnMgmt request being performed
+enum CONNMGMTREQTYPE {
+    // CM_CONNECT is used to connect to a libp2p peer
+    CM_CONNECT = 0;
```

Comment:
I don't think we need to add the CM here, if you look at the generated go code for example, the type information is already included in the exported symbol name.

Reply:
This is here to prevent namespace conflicts. If we ever need to use `CONNECT` as part of another enum, it will cause errors with other languages. This is a protocol buffer thing afaik

Reply:
Which documentation claims that?

Reply:
I can't recall it is a namespacing issue that occurs due to C of C++. 

If this enum has a field "CONNECT" and then we introduce another enum with a field CONNECT, there will be conflicts.

Reply:
https://stackoverflow.com/questions/29605650/how-to-using-namespace-in-protocol-buffers-with-c-sharp